### PR TITLE
Workshop submission with topic Spinning Wheel

### PR DESCRIPTION
**Pull request by HariOm987**
https://hackclub.slack.com/files/U019BA048CT/F01E7AH8K3K/workshop__recording.mp4

### CODE REVIEW

File: workshops/spinning_wheel/README.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
+---
+name: 'Spin The Wheel'
+description: 'Build a random picker spinning wheel by HTML, CSS and JavaScript'
+author: '@HariOm987'
+img: https://cloud-gvkfrv1qw.vercel.app/0image.png
+---
+You must have seen the spinning wheel by which we can select the random option, like a fortune wheel or a  lucky draw. But have you ever wondered how to make your own wheel by yourself? If you think that it is very tough then I am sorry, you are wrong. Just follow this workshop and you will have your customized spinning wheel in the next 20 minutes. You will surely accomplish this project with the help of this workshop and even if you are stuck at any point then instead of doing this 👇
```
Comment (sampoder) [inline workshops/spinning_wheel/README.md:7]: Check the grammar here

File: workshops/spinning_wheel/README.md [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+
+![person stuck in the problem GIF](https://cloud-b323sylf5.vercel.app/01439996693_lolwutaaa.gif)
+
+Just join [Hack Club Slack](https://hackclub.com/slack/) and ask your doubt in #code channel. There are thousands of high school students who can help you out. 😇
+
+Here's the [live demo](https://movie-selector.hariom04.repl.co/) and [final code](https://repl.it/@hariom04/movie-selector#index.html).
+
+![Final Output of our Workshop](https://cloud-4945tg5q1.vercel.app/0final.png)
+
+^^^ If you will follow this workshop strictly then you will make something like this, but feel free to add up your imaginations with patters and colors and you will end up with something better than this. So let's begin 😁✨
```
Comment (sampoder) [inline workshops/spinning_wheel/README.md:17]: spelling & phrasing

File: workshops/spinning_wheel/README.md [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+
+# Part 1 : Prerequisites
+* Account of Repl.it
```
Comment (sampoder) [inline workshops/spinning_wheel/README.md:20]: grammar

File: workshops/spinning_wheel/README.md [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+* Basic knowledge of HTML, CSS and JavaScript would be helpful but is not necessary to continue further.
+# Part 2 : Setup
+## Setting up your code environment
+We can use any code editor but keeping accessibility in the mind, I will be using Repl.it. It is an online code editor that is easy to use and can directly put your website to the real world.
```
Comment (sampoder) [inline workshops/spinning_wheel/README.md:24]: phrasing

**Review by sampoder — CHANGES_REQUESTED** [review]
like you asked, I checked it.. my main thing to look at would be to read this out loud and make sure the grammar / phrasing is very clear

cool workshop :D

File: workshops/spinning_wheel/README.md (hunk 3 above)
Comment (HariOm987) [reply workshops/spinning_wheel/README.md:20]: Okay, I think now I have corrected that. XD

File: workshops/spinning_wheel/README.md (hunk 1 above)
Comment (HariOm987) [reply workshops/spinning_wheel/README.md]: 👍

**Review by HariOm987 — COMMENTED** [review]
I have fixed the small changes. Now please review my workshop :)

**Comment by MatthewStanciu** [comment]
Hey @HariOm987, this is a great submission! I can totally imagine running a workshop like this in my club.

Content-wise, this is great. But I think it has a long way to go before it can be merged. There are many major formatting, grammar, punctuation, etc. errors throughout the workshop. I see errors like these all throughout the workshop, and it makes it difficult to read. Many are pretty small, but these all really add up. Here are some examples of major grammar errors I'm seeing (this is just a small sample):

- Throughout I see things like `Part X : description`, when instead it should be formatted like `Part X: Description`
- In some places, there's a lot of weird capitalization. For example, "The END"—it's unclear by "END" is in all caps. Same with "FILES" near the beginning of the workshop
- Weird formatting issues. For example, the list of files you see when you open first open the repl. "3. style.css" ends with a description, which looks like it shouldn't be on that line.
- "Building the Clock". What clock? I thought we're building a spinning wheel.
- "Congratulations, Now our Wheel is fully functional." is too long for a heading. Headings should generally not be full sentences
- None of the code blocks are highlighted in the correct languages

This has the potential to be a great workshop, and you did a lot of things right! (For example, I love the constant check-ins and code explanations—most workshop submissions don't have those). But I think it needs a few rounds of editing before it'll be ready to merge.

Do a deep dive in the workshop and see if you can fix all of instances of the things I mentioned above. Then, once you feel it's ready to re-review, open another pull request and ping me on Slack.

**Comment by HariOm987** [comment]
@MatthewStanciu Okay, Thanks for reviewing. I will surely do all the changes and will submit is again when I am confirmed that everything is alright.

**Pull request closed without merging**